include/cuco/detail/static_map/static_map_ref.inl

```diff
@@ -355,7 +268,7 @@ class operator_impl<
 
       auto const [state, intra_window_index] = [&]() {
         for (auto i = 0; i < window_size; ++i) {
-          switch (ref_.predicate_(window_slots[i], key)) {
+          switch (ref_.impl_.key_eq()(window_slots[i].first, key)) {
```

Comment:
That's what I'm worried about, we need the `key_eq` functor just for comparison and returning it by value meaning we copy sentinels every time we perform an equality check. That's really expensive.

Reply:
Fixed it in [36d1d5d](https://github.com/NVIDIA/cuCollections/pull/379/commits/36d1d5d2b51b40fd2b36efe6e451773c4c84a397)